datasets/io-land-cover/io_lulc.py

```diff
@@ -30,16 +45,22 @@
             "blob://ai4edataeuwest/io-lulc/io-lulc-model-001-"
             "v02-composite-v01-supercell-v02-clip-v01.geojson"
         ),
+        (
+            "blob://ai4edataeuwest/io-lulc/io-lulc-model-001-"
+            "v02-composite-v01-supercell-v02-clip-v01_2022_addition.geojson"
+        ),
```

Comment:
These are the same string, right?

```suggestion
        IO_LULC_9_CLASS_2022_ITEMS
```